app/ledger.js

```diff
@@ -1164,7 +1164,6 @@ var syncWriter = (path, obj, options, cb) => {
     if (options.noRetryP) return
 
     options.noRetryP = true
-    console.log('\nretrying ' + path)
```

Comment:
for follow-up changes in a PR, you can just commit, rebase -i, then force-push, so there is no extra rebase step necessary when review is done